server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -73,19 +73,18 @@ pub struct RemoteDataSynchroniser {
 
 impl RemoteDataSynchroniser {
     /// Request initialisation
-    /// TODO improve logic for initialisation check, requires central changes as per this issue:
-    /// https://github.com/openmsupply/open-msupply-internal/issues/30#issuecomment-1567639141
-    ///
-    /// This api call is blocking, and will wait while central server adds relevant records to
-    /// sync queue. If there is a connection problem while intialisaion is happenning, we don't
-    /// want to restart initialisation on next sync, thus polling is introduced.
-    /// When initialisaiton is in progress api will return sync_is_running
+    /// This api call is blocking, and will wait while central server adds relevant records to sync queue.
+    /// If there is a connection problem while intialisaion is happenning, we don't want to restart initialisation on next sync
+    /// thus polling and check for initialisation status is introduced.
+    /// When initialisaiton is in progress api will return sync_is_running and get_site_info will return initalisationStatus = started
```

Comment:
:nit
```suggestion
    /// This api call is blocking, and will wait while central server adds relevant records to sync queue.
    /// If there is a connection problem while initialisation is happening, we don't want to restart initialisation on next sync
    /// thus polling and a check for initialisation status was introduced. When initialisation is in progress,
    /// api will return `sync_is_running` and `get_site_info` will return `initialisationStatus` = `started`
```

Reply:
> api will return `"sync_is_running"`

It should return code `initialisation_in_progress` now https://github.com/openmsupply/msupply/pull/13111/files#diff-6e86c489c506a78643b0a626fac2e99f69f7ec8e649ff144c9e8284b3391f9e4

Reply:
thanks @mark-prins for suggestion

Reply:
thanks @Chris-Petty, changed. And also did another test, by putting breakpoint in sync_create_records_for_site, then starting initialisation, then stopping and restarting remote server, and got an error, then after this change 7e1dfeb0af96f19d8278e490c3dabacc78eedbf9 no error (it restarts awaiting initialisation), nice catch.

Ideally:

https://github.com/openmsupply/open-msupply/blob/7e1dfeb0af96f19d8278e490c3dabacc78eedbf9/server/service/src/sync/api/get_site_status.rs#L14-L19

Would Deserialise to:

```
pub(crate) enum SiteStatusCodeV5 {
    SyncIsRunning,
    IntegrationInProgress,
    InitialisationInProgress,
    Idle,
    Other(String)
}
```

Notice Other(String), but that would be a bit too much for work this PR